### ISSUE TITLE
Loaded BackendUser in frontend context causes 

### ISSUE THREAD

**Issue by dmolineus** [menatwork/syncCto#247]
After installing synccto I get following error in the frontend `mysqli::real_escape_string(): Couldn't fetch mysqli`. As far as I could debug it's caused because the BackendUser is insantiated in the Helper class which is triggered during the `SyncCtoSubscriber` event initialization.

Because other extensions create the database connection before synccto the Contao object stack order get screwed up.

I don't know how to solve it. The easiest way to avoid it is to not register any listener in the frontend context at all. So my question is, if any event is required in the frontend context?

**Comment by stefanheimes** [comment]
I agree it is a bug. We only need this for the backend. I will check it.

**Comment by stefanheimes** [comment]
I have change some class calls and remove some imports. This should solve the problem.

**Comment by dmolineus** [comment]
Wanted to test it in the current project where the issue raised. Unfortunately ctoCommunication depends on guzzlehttp/guzzle:~5.3. In this project guzzle v6 is used so I can't confirm the problem got solved right now. Have to setup a test case for it instead.

Any plans to upgrade to guzzle 6 soon?

**Comment by stefanheimes** [comment]
Since the current Contao LTS has as requirement php 5.4, i see no problem to use the new guzzle. Unfortunately i have to rewrite the communication class.
